scour/scour.py

```diff
@@ -2880,6 +2880,14 @@ def scourString(in_string, options=None):
    global numBytesSavedInTransforms
    doc = xml.dom.minidom.parseString(in_string)
 
+   # determine number of flowRoot elements in input document
+   # flowRoot elements don't render at all on current browsers (04/2016)
+   cnt_flowText_el = len(doc.getElementsByTagName('flowRoot'))
+   if cnt_flowText_el:
+      print("SVG input document uses {} flow text elements, which won't render on browsers!".format(cnt_flowText_el))
+      if options.error_on_flowtext:
+         sys.exit(1)
+
```

Comment:
Right now it does not really matter (as it seems we don't touch `flowRoot`s, even if they are empty) but in general I guess "compatibility checks" should be done somewhere at the end of `scourString()` as we most likely want to check Scours output for compatibility, not the input?